Update tenant-rename.md (#2885)
MicrosoftDocs/OfficeDocs-SharePoint: SharePoint/SharePointOnline/tenant-rename.md

2. Go to the [Custom domain names page in the Azure AD admin center](https://aka.ms/SPORenameAddDomain).

    > [!IMPORTANT]

>     > Use the hyperlink [https://aka.ms/SPORenameAddDomain](https://aka.ms/SPORenameAddDomain) provided above. Failiure to use this hyperlink will prevent you from successfully adding a custom domain.

Reviewer: ```suggestion
    > You must use the hyperlink [https://aka.ms/SPORenameAddDomain](https://aka.ms/SPORenameAddDomain). Failure to use this hyperlink will prevent you from successfully adding a custom domain.
```

3. Select **Add custom domain**.

4. In the **Custom domain name** box, add the full new “.onmicrosoft.com” domain, and then select **Add domain**.

    ![Custom domain name pane](media/custom-domain-name.png)

    > [!IMPORTANT]

>     > When adding the domain this needs to be a “onmicrosoft.com” domain. For example, if you want to rename your tenant to fabrikam.sharepoint.com, the domain that you enter should be fabrikam.onmicrosoft.com. Adding a “onmicrosoft.com” domain does not need to be purchased or require any public DNS registration.

Reviewer: ```suggestion
    > When adding the domain, it needs to be an “onmicrosoft.com” domain. For example, if you want to rename your tenant to fabrikam.sharepoint.com, the domain that you enter should be fabrikam.onmicrosoft.com. The “onmicrosoft.com” domain does not need to be purchased to be added and does not require any public DNS registration.
```

 
5. Make sure you get a confirmation message. If the domain isn't available, try a different domain. 
 
6. After getting a confirmation that the domain was added successfully, you might see a message that the properties could not be found. Select the message to refresh domain references.

    > [!WARNING]
    > Do NOT add any other domains.
    > Do NOT configure the new domain as the initial domain.

>     > If after adding the domain you are prompted to create a new TXT record with your domain name registrar, then the domain has NOT been added correctly and you will NOT be able to perform a rename. If you are prompted then you will need to delete the invalid domain and return to Step 2 above.

Reviewer: ```suggestion
    > If, after adding the domain, you are prompted to create a new TXT record with your domain name registrar, then the domain has NOT been added correctly and you will NOT be able to perform a rename. If you are prompted, you will need to delete the invalid domain and return to the previous Step 2.
```

 
7. Confirm that your domain has been added to the list. 

    ![Domain status verified](media/domain-verified.png)

    > [!IMPORTANT]

>     > The domain name must have the status of "Verified". If the status is NOT "Verified" then you will NOT be able to perform a rename. 

Reviewer: ```suggestion
    > The domain name must have the status of "Verified". If the status is NOT "Verified", you will NOT be able to perform a rename. 
```

    `Start-SPOTenantRename -DomainName "fabrikam" -ScheduledDateTime "2021-12-31T10:25:00"`

    > [!NOTE]

>     > If the PowerShell command Start-SPOTenantRename is not found or nothing is returned, then ensure that you have installed the latest SharePoint Online Management Shell. If required, before installing the latest version you may need to uninstall all previous versions by running `Uninstall-Module Microsoft.Online.SharePoint.PowerShell -Force -AllVersions` 

Reviewer: ```suggestion
    > If the PowerShell command Start-SPOTenantRename is not found or nothing is returned, ensure that you have installed the latest SharePoint Online Management Shell. If required, before installing the latest version, you might need to uninstall all previous versions by running `Uninstall-Module Microsoft.Online.SharePoint.PowerShell -Force -AllVersions`.
```
